Assets/Scripts/Interfaces/IDamageable.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Interfaces
+{
+    public interface IDamageable
```

Comment:
Would an abstract class be useful here? I mean enemies and the player both share health and the "loosing health behavior".